ios/browser/api/ads/brave_ads.mm

```diff
@@ -1528,9 +1535,7 @@ - (void)recordP2AEvents:(const std::vector<std::string>&)events {
 }
 
 - (std::optional<base::Value>)getProfilePref:(const std::string&)path {
-  if (path == brave_news::prefs::kBraveNewsOptedIn ||
-      path == brave_news::prefs::kNewTabPageShowToday ||
-      path == ntp_background_images::prefs::kNewTabPageShowBackgroundImage ||
+  if (path == ntp_background_images::prefs::kNewTabPageShowBackgroundImage ||
```

Comment:
i'm not sure i understand what is going on here, but if a user enables "show background image" but disables "show sponsored image", ads should not be enabled?

Reply:
basically i just want to check if a user disables ads everywhere and enables "show background image" (but not "show sponsored images"), no requests will be made to the ads service

Reply:
Setting these preferences to true for iOS is no-op for now and won't lead to any extra  network requests.
We will fix these preferences as a follow-up within this ticket: https://github.com/brave/brave-browser/issues/39658